hub/powertoys/run.md

```diff
@@ -87,13 +87,25 @@ With PowerToys v0.31 and on, there are system level actions you can now execute.
   | `Hibernate` | Hibernates the computer |
   | `Empty Recycle Bin` | Empties the recycle bin |
 
-## Indexer settings
+## Plugin manager
 
-If indexer settings are not set to cover all drives, you will receive the following warning:
+With PowerToys v0.33 and on, the PowerToys Run settings menu includes a plugin manager that allows you to enable/disable the various plugins currently available. By clicking and expanding the various sections, you can customize the activation phrases used by each plugin. In addition you can select whether a plugin appears in global results, as well as set additional plugin options where available. 
+
+## Program parameters
+
+When starting a program with the PowerToys Run program plugin, you can specify parameters for the program to use by utilizing the following command formats:
+- `program -param ...`
+- `program /param ...`
+- `program --parameter ...`
+- `program -- parameter ...` (Here `--` is used as delimiter by PT Run.)
+
```

Comment:
This parameter commands section feels a bit confusing to me. Could you provide an example of one of these parameter commands being used as a demonstration?

Reply:
agreed, this should be a real world example